tests/operators/test_trigger_dagrun.py

```diff
@@ -41,6 +40,7 @@
 pytestmark = pytest.mark.db_test
 
 DEFAULT_DATE = datetime(2019, 1, 1, tzinfo=timezone.utc)
+DEFAULT_RUN_ID = f"{DagRunType.MANUAL}__{DEFAULT_DATE.isoformat()}"
```

Comment:
Should we add a case that contains random string (as the logical_date could be none)

Reply:
Probably doesn’t matter since we don’t parse the run ID, this is just a string to Airflow. Maybe let’s just use a string like `test_run_id` instead.